apps/web/src/components/ai/DataAgentChatAccessibility.tsx

```
@@ -0,0 +1,618 @@
+        return null;
+    }
+  };
+
+  if (!data || data.length === 0) return null;
+
+  return (
+    <Card 
+      className="mt-3"
+      ref={cardRef}
+      role="region"
+      aria-labelledby={`${testId}-title`}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)